### DISCUSSION TITLE
Flashing MCU on 4 Plus

### DISCUSSION THREAD

**Discussion by hopkinsj09** [OpenNeptune3D/OpenNept4une#177] [General]
Just bought another, but the 4 plus this time. When flashing the MCU, there isn't a micro USB slot to flash the physical MCU, so I skipped that step. When checking the MCU versions in Fluidd's system tab, it is v0.12.xxx. Is this not needed on the plus, or need to do it differently? Everything on Google about updating firmware is via the touchpad, but this thing never was powered on with the stock Elegoo firmware, and I do not really need the touchpad.

thanks!
Joe

**Comment by phillipthelen** [comment]
The SD card slot for updating the MCU is hidden in the newer printers. You can find instructions for accessing it here: https://github.com/OpenNeptune3D/OpenNept4une/wiki/Hidden-microSD-Slot-Access

you should still update the MCU and keep it up to date with klipper for best performance

**Reply by hopkinsj09** [reply]
Thanks! I feel dumb for not finding this.

Yeah I'm totally going to update it, just couldn't find the slot. Still waiting on a few things to show up before even starting to print.

**Reply by halfmanbear** [reply]
The version you would have seen with v0.12 would have been the secondary virtual MCU if you check that page again there will be another listed

**Reply by hopkinsj09** [reply]
found it! I appreciate the project, guys!
